docs/Features-for-Go-in-VS-Code.md

```diff
@@ -4,20 +4,23 @@
 - Signature Help for functions as you type (using `gogetdoc` or `godef`+`godoc`)
 - Quick Info on the symbol as you hover over it (using `gogetdoc` or `godef`+`godoc`)
 
+If `go.useLanguageServer` is enabled, most of the above features will be
+offered through the Go language server `gopls`.
+
 ### Code Navigation
 
-- Go to or Peek Definition of symbols (using `gogetdoc` or `godef`+`godoc`)
-- Find References of symbols and Implementations of interfaces (using `guru`)
-- Go to symbol in file or see the file outline (using `go-outline`)
-- Go to symbol in workspace (using `go-symbols`)
+- Go to or Peek Definition of symbols (using `gogetdoc` or `godef`+`godoc` or `gopls`)
```

Comment:
How come such addition is not done in the `IntelliSense` section?

Reply:
I changed IntelliSense section to just explicitly mention `gopls` for each entry.